src/style.css

```diff
@@ -1,3 +1,124 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
 }
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+  background: hsl(0, 0%, 100%);
+  border: 1px solid #f3f3f3 ;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__computer {
+  display: flex;
+  justify-content: center;
+  padding-bottom: 40px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+}
+
+.card__imac {
+  color: hsl(234, 80%, 12%);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  padding-bottom: 4px;
+}
+
+.card__product-code {
+  font-weight: 400;
+  font-size: 10px;
+  padding-bottom: 16px;
+  color: hsl(244, 8%, 41%);
+}
+
+.card_star-reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card_reviews {
+  width: 53px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  text-align: right;
+  padding-bottom: 24px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 16px;
+}
+
+.card__price-name {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: rgba(97, 96, 112, 1);
+}
+
+.card__price-sum {
+  font-weight: 700;
+  font-size: 16px;
+  text-align: right;
+}
+
+.card_link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
+  width: 166px;
+  height: 40px;
+  background-color: #00acdc;
+  border-radius: 5px;
+
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  color: hsl(0, 0%, 100%);
+}
+
+.card_link:hover {
+  background-color: hsl(0, 0%, 100%);
+  color: #00acdc;
+  border: 1px solid #00acdc ;
+}
+
+.stars {
+  width: 96px;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  padding-bottom: 24px;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
can be minimized to just background shorthand property
https://www.w3schools.com/css/css_background_shorthand.asp